backend/api/src/handlers/types/handler-env-var.ts

```diff
@@ -0,0 +1 @@
+export type EnvironmentVariable = "CODE_BLOCK_TABLE_NAME";
```

Comment:
Is the idea that this will be a union template literal type? Mind adding something in the commit message so it's a bit more clear what's going on here?